.github/workflows/create_pull_request.yml

```diff
@@ -32,4 +32,24 @@ jobs:
             head: "${{ github.event.client_payload.ReleaseBranchName }}-docs",
             base: "${{ github.event.client_payload.PullRequestBase }}",
             body: `${{ github.event.client_payload.PullRequestBody }}`
-          });
\ No newline at end of file
+          });
+
+    - name: Get pull request number for ${{ github.event.client_payload.ReleaseBranchName }}-docs
```

Comment:
Why do we need the separate step to get pull-request number?
I think previous step `github.pulls.create` should return object with number:
https://docs.github.com/en/rest/reference/pulls#create-a-pull-request

Reply:
Reworked, thank you!